cabot/cabotapp/models.py

```diff
@@ -859,7 +872,14 @@ def _run(self):
             result.error = str(e)
             result.succeeded = False
 
-        return result
+        return result, []
+
+
+class StatusCheckResultTag(models.Model):
+    value = models.CharField(max_length=255, blank=False, primary_key=True)
```

Comment:
Should we add a created/updated time to the model so we can delete old values more easily? 

Also do we want this to be the primary key or just unique? All the other models have an autoincremented id as the primary key...but I don't really see a compelling reason why we'd need that here so I suppose this being the primary key seems better

Reply:
ok nvm on the first comment, cleaning up orphaned tags from StatusCheckResults like you have makes more sense